src/physics.ts

```diff
@@ -461,12 +480,18 @@ export namespace physics {
             let collision = false;
 
             for (let i = world.dynamicBodies.length; i--;) {
+
+                // Only moving objects can collide if they didn't last worldStep
+                const bodyI = world.dynamicBodies[i];
+                if (!bodyI.velocity) {
```

Comment:
I'm going to merge this and then remove these bits, it'll work for scorched turf because there aren't many bodies, but in a complex system resolving penetration of bodies can result in new collisions with bodies that didn't previously collide